ProcessMaker/processmaker
# Add FileDownload component for display form type

This PR fixes an issue with the FileDownload component for display type forms.

How to test:
Drag a FileDownload component into the form builder in a display form type. The FileDownload should render and you should not see an `unknown custom element "FileDownload" ...` error in the console anymore.
